src/EnergyPlus/UnitarySystem.cc

```diff
@@ -3369,6 +3405,24 @@ namespace UnitarySystems {
                             setSystemParams(state, thisSys, TotalFloorAreaOnAirLoop, thisObjectName);
                             // The Node was found among the exhaust nodes, now check that a matching inlet node exists
                             ZoneInletNodeFound = searchZoneInletNodesByEquipmentIndex(state, thisSys.ATMixerOutNode, thisSys.ControlZoneNum);
+                        } else {
+                            int ZoneInletNum = 0;
+                            ZoneInletNodeFound = searchZoneInletNodes(state, thisSys.ATMixerOutNode, thisSys.ControlZoneNum, ZoneInletNum);
+                            if (ZoneInletNodeFound) {
+                                for (int NodeNum = 1; NodeNum <= state.dataZoneEquip->ZoneEquipConfig(thisSys.ControlZoneNum).NumReturnNodes;
+                                     ++NodeNum) {
+                                    InducedNodeFound = ZonePlenum::ValidateInducedNode(
+                                        state,
+                                        thisSys.AirInNode,
+                                        state.dataZoneEquip->ZoneEquipConfig(thisSys.ControlZoneNum).ReturnNode(NodeNum));
+                                    if (InducedNodeFound) break;
+                                }
+                                if (InducedNodeFound) {
+                                    thisSys.m_ZoneInletNode = thisSys.ATMixerOutNode;
+                                    ZoneEquipmentFound = true;
+                                    setSystemParams(state, thisSys, TotalFloorAreaOnAirLoop, thisObjectName);
+                                }
+                            }
```

Comment:
Allow Unitary to accept an induced node of return plenum as an inlet node with ATMixer at the supply side, in addition to a zone exhaust node.